stable/rabbitmq-ha/values.yaml

```diff
@@ -223,3 +223,24 @@ serviceAccount:
   ## The name of the ServiceAccount to use.
   ## If not set and create is true, a name is generated using the fullname template
   # name:
+
+secret:
+  autogenerate: true
+
+prometheus:
+  alerts:
+    enabled: true
+    selector:
+      role: alert-rules
+    labels: {}
+  exporter:
```

Comment:
I suggest removing the alerts section and flatten a bit as suggested in the chart best practices:
```
prometheusExporter:
  enabled: true
  [...]
```